tests/functional/test_show.py

```diff
@@ -151,6 +152,7 @@ def test_pip_show_is_short(script):
     assert len(lines) <= 10
 
 
+@pytest.mark.network
```

Comment:
Oh sorry, I did not spot it was on 2 separate tests.
Ideally, the change to `test_pip_show_divider` should not happen in this PR as it is unrelated...